packages/perseus/src/widgets/interactive-graphs/graphs/sinusoid.tsx

```diff
@@ -0,0 +1,107 @@
+import {color} from "@khanacademy/wonder-blocks-tokens";
+import {Plot} from "mafs";
+import * as React from "react";
+
+import {movePoint} from "../reducer/interactive-graph-action";
+
+import {StyledMovablePoint} from "./components/movable-point";
+
+import type {Coord} from "../../../interactive2/types";
+import type {SinusoidGraphState, MafsGraphProps} from "../types";
+import type {vec} from "mafs";
+
+type SinusoidGraphProps = MafsGraphProps<SinusoidGraphState>;
+
+export type SineCoefficient = [
+    number, // amplitude
+    number, // angularFrequency
+    number, // phase
+    number, // verticalOffset
+];
```

Comment:
A 4-tuple is a lot for me to hold in my head. Would it be easier if this were an object with named fields?

```ts
export type SineCoefficient = {
  amplitude: number,
  angularFrequency: number,
  phase: number,
  verticalOffset: number,
}
```

`computeSine` could take this whole object as a param and destructure it internally.